joint_trajectory_controller/test/test_trajectory_controller_utils.hpp

```diff
@@ -363,7 +365,13 @@ class TrajectoryControllerTest : public ::testing::Test
     const auto end_time = start_time + wait_time;
     while (clock.now() < end_time)
     {
-      traj_controller_->update(clock.now(), clock.now() - start_time);
+      // TODO(christophfroehlich): use the node's clock here for internal comparison
+      // if using RCL_STEADY_TIME ->
+      //   C++ exception with description
+      //   "can't compare times with different time sources" thrown in the test body.
+      // traj_controller_->update(clock.now(), clock.now() - start_time);
```

Comment:
should we create the node_ with `NodeOptions`, `clock_type = RCL_STEADY_TIME`?

Reply:
Some of these may be from a time we didn't have any clock solution, worth reassessing if needed